src/types/transaction.ts

```diff
@@ -68,7 +78,7 @@ export function toEthTx(
     input: txJSON.data!,
     nonce: txJSON.nonce!,
     to: transaction.to?.toString() ?? null,
-    transactionIndex: index,
+    transactionIndex: bigIntToHex(BigInt(index)),
```

Comment:
check also if undefined?

Reply:
BigInt("") := 0n

And apibara types this index as a string not a string | undefined no?